src/lib/components/FileUploader/FileUploader.js

```diff
@@ -142,6 +150,10 @@ export const FileUploaderComponent = ({
   const displayImportBtn =
     filesEnabled && isDraftRecord && hasParentRecord && !filesList.length;
 
+  const draftFilesWarning =
+    i18next.t(fileAreaWarning) ||
```

Comment:
I am not sure I understand the `||`: the `prop` will always have a value, the passed string or the default translated string.
Shouldn't we have simply `const draftFilesWarning = fileAreaWarning`?
Same for the other one above.

Reply:
I've added this s.t. passing a falsy value (empty string or null) also results in the default value, because that made the configuration in Invenio-App-RDM a little bit easier.
Otherwise, to display the default message, we would have to make sure that this prop isn't passed into the `FileUploader` on the deposit form: https://github.com/inveniosoftware/invenio-app-rdm/pull/1436/files#diff-b5e1b966b44dd2d7d0fc9688d8243e176d2a3c1242200e495b2bb4598d60be4bR167

If that's the preferred way, I can change it of course.

Reply:
I see now thanks for the explanations! :) I am for more conventional solutions e.g.
```
DEFAULT_DRAFT_FILES_WARNING = i18next.t('File addition...


export const FileUploaderComponent = ({
    ....
    customfileAreaWarning
    ...
...uiProps
}) => {
   
    const fileAreaWarning = customfileAreaWarning || DEFAULT_DRAFT_FILES_WARNING;
    ...
};

 FileUploaderComponent.defaultProps = {
    ...
    fileAreaWarning: undefined,
};
```
but I am also ok with this.

@zzacharo @kpsherva opinions/preferences?

Reply:
I am ok with defaultProps, since it seems to me as more "react" way to set the default